app/views/items/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Item Information</h1>
+<%= link_to "#{@user.first_name} #{@user.last_name}'s item", user_path(@user) %>
+<ul>
+  <li>Name: <%= "#{@item.name}" %></li>
+  <li>Description: <%= "#{@item.description}" %></li>
+  <li>Size: <%= "#{@item.size}" %></li>
+  <li>Note: <%= "#{@item.note}" %></li>
+</ul>
+
+<% if @authorized_user %>
+<%= link_to 'Edit Item', edit_group_list_item_path(@group, @list, @item) %>
+<%= link_to 'Delete Item', group_list_item_path(@group, @list, @item), method: :delete, data: { confirm: 'Are you sure?' } %>
```

Comment:
could use some string interpolation to improve the "Are you sure?" message

Reply:
Added this functionality.